src/webportal/src/app/user/fabric/user-profile/ssh-list-dialog.jsx

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+import {
+  DefaultButton,
+  PrimaryButton,
+  DialogType,
+  Dialog,
+  DialogFooter,
+  TextField,
+} from 'office-ui-fabric-react';
+
+import t from '../../../components/tachyons.scss';
+
+const SSHListDialog = ({ onDismiss, onAddPublickeys }) => {
+  const [error, setError] = useState('');
+  const [inputTitleError, setInputTitleError] = useState('');
+  const [inputValueError, setInputValueError] = useState('');
+  const [processing, setProcessing] = useState(false);
+  const [title, setTitle] = useState('');
+  const [value, setValue] = useState('');
+
+  const onAddAsync = async () => {
+    if (title.trim() === '') {
+      setInputTitleError('Please input title');
+    } else if (value.trim() === '') {
+      setInputValueError('Please input ssk value');
+    } else {
```

Comment:
Please validate value matches SSH key's format.

You can use this regex: `/^ssh-rsa AAAA[0-9A-Za-z+/]+[=]{0,3}.*$/`

Reply:
done